src/transformers/models/marian/configuration_marian.py

```diff
@@ -160,3 +166,226 @@ def __init__(
             forced_eos_token_id=forced_eos_token_id,
             **kwargs,
         )
+
+
+# Copied from transformers.models.bart.configuration_bart.BartOnnxConfig with Bart->Marian
```

Comment:
Since the Marian model is copied from BART (see `modeling_marian.py`), I adopted a similar approach for the ONNX config.

Reply:
Yes, nice!